src/modules/hip/hip_advanced_augmentation.cpp

```diff
@@ -0,0 +1,924 @@
+                                                                     tensor_info._in_channels,
+                                                                     handle_obj->mem.mgpu.inc,
+                                                                     handle_obj->mem.mgpu.dstInc,
+                                                                     in_plnpkdind,
+                                                                     out_plnpkdind);
+
+#elif defined(STATIC)
+        hip_exec_color_cast_batch(srcPtr, dstPtr, handle, tensor_info, in_plnpkdind, out_plnpkdind, max_height, max_width);
+#endif
+    return RPP_SUCCESS;
+}
+
+RppStatus
+color_cast_hip_batch_tensor_fp16(Rpp16f *srcPtr, Rpp16f *dstPtr, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
+{
+//     int in_plnpkdind = getplnpkdind(tensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
+//     InitHandle *handle_obj = handle.GetInitHandle();
+//     Rpp32u max_height, max_width;
+//     max_size(handle_obj->mem.mgpu.csrcSize.height, handle_obj->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
+// #if defined (HIPRTC)
+// #elif defined(STATIC)
+//      hip_exec_color_cast_batch_tensor_fp16(srcPtr1, srcPtr2, dstPtr, handle, in_plnpkdind, out_plnpkdind, max_height, max_width);
+// #endif
+    return RPP_SUCCESS;
+}
+
+RppStatus
+color_cast_hip_batch_tensor_fp32(Rpp32f *srcPtr, Rpp32f *dstPtr, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
+{
+    int in_plnpkdind = getplnpkdind(tensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
+    InitHandle *handle_obj = handle.GetInitHandle();
+    Rpp32u max_height, max_width;
+    max_size(handle_obj->mem.mgpu.csrcSize.height, handle_obj->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
+
+    #if defined (HIPRTC)
+
+    #elif defined(STATIC)
+
+        hip_exec_color_cast_batch_fp32(srcPtr, dstPtr, handle, tensor_info, in_plnpkdind, out_plnpkdind, max_height, max_width);
+
+    #endif
+
+    return RPP_SUCCESS;
+}
+
+RppStatus
+color_cast_hip_batch_tensor_int8(Rpp8s *srcPtr, Rpp8s *dstPtr, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
+{
+    int in_plnpkdind = getplnpkdind(tensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
+    InitHandle *handle_obj = handle.GetInitHandle();
+    Rpp32u max_height, max_width;
+    max_size(handle_obj->mem.mgpu.csrcSize.height, handle_obj->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
+
+    #if defined (HIPRTC)
+
+    #elif defined(STATIC)
+
+        hip_exec_color_cast_batch_int8(srcPtr, dstPtr, handle, tensor_info, in_plnpkdind, out_plnpkdind, max_height, max_width);
+
+    #endif
+
+    return RPP_SUCCESS;
+}
+/******************** lut ********************/
+
+// RppStatus
+// lut_hip_batch(Rpp8u* srcPtr, Rpp8u* dstPtr, Rpp8u* lut, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
```

Comment:
Remove the commented lut_hip_batch function, Also remove the 6 other *_hip_batch functions. Let's just have the *_hip_batch_tensor for these 7 functions.

Reply:
done